x/cronos/keeper/evm_log_handlers_test.go

```diff
@@ -297,3 +297,82 @@ func (suite *KeeperTestSuite) TestSendToIbcHandler() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestSendCroToIbcHandler() {
+	contract := common.BigToAddress(big.NewInt(1))
+	var data []byte
+
+	testCases := []struct {
+		msg       string
+		malleate  func()
+		postcheck func()
+		error     error
+	}{
+		{
+			"not enough balance, fail",
+			func() {
+				coin := sdk.NewCoin(suite.evmParam.EvmDenom, sdk.NewInt(10000000000000))
+				input, err := keeper.SendCroToIbcEvent.Inputs.Pack(
+					"recipient",
+					coin.Amount.BigInt(),
+				)
+				suite.Require().NoError(err)
+				data = input
+			},
+			func() {},
+			errors.New("0aphoton is smaller than 10000000000000aphoton: insufficient funds"),
+		},
+		{
+			"success send cro to ibc",
+			func() {
+				suite.app.CronosKeeper.SetExternalContractForDenom(suite.ctx, suite.evmParam.EvmDenom, contract)
+				coin := sdk.NewCoin(suite.evmParam.EvmDenom, sdk.NewInt(12300000000))
+				err := suite.MintCoins(sdk.AccAddress(contract.Bytes()), sdk.NewCoins(coin))
+				suite.Require().NoError(err)
+
+				balance := suite.app.BankKeeper.GetBalance(suite.ctx, sdk.AccAddress(contract.Bytes()), suite.evmParam.EvmDenom)
+				suite.Require().Equal(coin, balance)
+
+				// Mint coin for the module
+				suite.MintCoinsToModule(types.ModuleName, sdk.NewCoins(sdk.NewCoin(types.IbcCroDenomDefaultValue, sdk.NewInt(123))))
+
+				err = suite.MintCoins(sdk.AccAddress(contract.Bytes()), sdk.NewCoins(coin))
+				suite.Require().NoError(err)
+
+				input, err := keeper.SendToIbcEvent.Inputs.Pack(
+					"recipient",
+					coin.Amount.BigInt(),
+				)
+				data = input
+			},
+			func() {},
```

Comment:
I think it is worth doing the post checking that:
- balance of the contract is good
- balance of the module account is good